src/librustc_target/spec/aarch64_apple_darwin.rs

```diff
@@ -0,0 +1,32 @@
+use crate::spec::{LinkerFlavor, Target, TargetOptions, TargetResult};
+
+pub fn target() -> TargetResult {
+    let mut base = super::apple_base::opts();
+    base.cpu = "apple-a12".to_string();
```

Comment:
The developer kit [ships with an A12Z](https://en.wikipedia.org/wiki/Developer_Transition_Kit_(2020)#Specifications). 

> A12Z Bionic is the same as the A12X chip, but with an extra GPU core enabled
>
> - [Wikipedia](https://en.wikipedia.org/wiki/Apple_A12Z)

> The A12X is an 8-core variant of the A12 
> 
> - [Wikipedia](https://en.wikipedia.org/wiki/Apple_A12X)

I saw nothing closer in LLVM. We should also evaluate whatever chip ships in the first commercial hardware release.

Reply:
This brings up an interesting question though – should we actually strive to leave this set to `apple-a12` to support these soon-legacy devices?

Reply:
I think the answer is **no**. The DTK needs to be returned to Apple, so any one that has one is a curiosity, not someone we have to worry about as a prime supported target.